tests/unit/detection/test_detection_dataset.py

```diff
@@ -221,6 +223,40 @@ def test_detection_dataset_init_train_pct(
     assert len(data.train_ds) == 24
 
 
+def test_detection_dataset_verify(basic_detection_dataset):
```

Comment:
ideally this unit test should check that the warning is called if image resolution is too high, but i think this is ok for now